bundles/org.openhab.voice.voicerss/src/main/resources/OH-INF/i18n/voicerss.properties

```diff
@@ -0,0 +1,8 @@
+# service
+
+service.voice.voicerss.label = VoiceRSS Text-to-Speech
+
+# bundle config
+
+voice.config.voicerss.apiKey.label = VoiceRSS API Key
+voice.config.voicerss.apiKey.description = The API Key to get access to http://www.voicerss.org. You need to register with at least a free account to get an API key.
```

Comment:
The project still doesn't have secured HTTP protocol on their website. That's freaking.